packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -6,6 +6,11 @@ export interface ChipProps extends PropsWithChildren {
    */
   readonly ariaLabel?: string;
 
+  /**
+   * The testing id for the chip if necessary. Defaults to `chip-wrapper`.
+   */
+  dataTestID?: string;
```

Comment:
I think it may be worth changing this to `testID` to match other test ID props we expose such as 
https://github.com/GetJobber/atlantis/blob/JOB-79340/JOB-80485/actually-separate-third-go/packages/components/src/Icon/Icon.tsx#L31

Reply:
good catch, yeah that makes sense to me